Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputManager.h

```diff
@@ -39,6 +39,17 @@ namespace AzToolsFramework
         Q_OBJECT
 
     public:
+        enum CursorInputMode {
+            CURSOR_MODE_NONE,
+            CURSOR_MODE_CAPTURED,   //< Sets whether or not the cursor should be constrained to the source widget and invisible.
+                                    //< Internally, this will reset the cursor position after each move event to ensure movement
+                                    //< events don't allow the cursor to escape. This can be used for typical camera controls
+                                    //< like a dolly or rotation, where mouse movement is important but cursor location is not.
+            CURSOR_MODE_WRAPPED,    //< Flags whether the curser is going to wrap around the soruce widget.
+            CURSOR_MODE_WRAPPED_X,
+            CURSOR_MODE_WRAPPED_Y 
+        };
```

Comment:
update: We should use `enum class` here, also enums use `UpperCamelCase` convention, now `ALL_CAPS` (that is reserved for macros)

Reply:
nit: For doxygen to pick up the api comments you also need `//!<` (missing exclamation mark)